.github/workflows/update_testrail_results.yml

```diff
@@ -16,6 +16,12 @@ on:
       testRail_name:
         type: string
         default: ''
+      test_specific_OS_needed:
+        type: boolean
+        default: false
+      test_OS_name:
+        type: string
+        default: ''
```

Comment:
I don't think you need two inputs for this. You should be able to just use `test_OS_name` and reference that directly since it already has a default value